ui/tests/unit/serializers/service-test.js

```diff
@@ -0,0 +1,23 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Serializer | service', function (hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
+  test('it exists', function (assert) {
+    let store = this.owner.lookup('service:store');
+    let serializer = store.serializerFor('service');
+
+    assert.ok(serializer);
+  });
+
+  test('it serializes records', function (assert) {
+    let store = this.owner.lookup('service:store');
+    let record = store.createRecord('service', {});
+
+    let serializedRecord = record.serialize();
+
+    assert.ok(serializedRecord);
```

Comment:
`question`: Is this generated via `ember-cli` or are you trying to design this like a kata?

Reply:
this is auto-generated as a default ember blueprint